### ISSUE TITLE
make doctrine.cli entity-manager configurable with ENV

### ISSUE THREAD

**Issue by lenada** [doctrine/DoctrineORMModule#367]
Setting up two entity managers  with different connections / orm configurations works as expected.

#### Problem
When using the `doctrine-module` cli, `doctrine.entitymanager.orm_default` is "hardcoded" in `DoctrineORMModule\Module` and therefore only allows to use it with the default entity manager.

#### Solution
*use environment variable to use something other then `orm_default`.*
There may be other approaches - would be happy to hear about them

```php
  $ormDefault = (getenv('ORM_DEFAULT') ? getenv('ORM_DEFAULT') : 'orm_default');
  // [...]
  $entityManager = $serviceLocator->get('doctrine.entitymanager.' . $ormDefault);
```
If that kind of approach seems feasible , I would be happy to do a PR.

to be discussed?:
 - ORM_DEFAULT naming for env
 - ($foo ? $foo: 'bar' ) style

**Comment by Ocramius** [comment]
This is a duplicate of #97 - requires a major overhaul that is actually a BC break right now.

**Comment by lenada** [comment]
@Ocramius sorry for the duplicate. thanks for pointing me there!

**Comment by Ocramius** [comment]
@lenada no big deal. The issue is almost solved in 1.0.x, but it's far from merging right no, mainly because of lack of time.
